src/components/Map/Map.js

```diff
@@ -24,29 +24,35 @@ function Map({
   const history = useHistory();
   const [content, setContent] = useState('');
 
-  const goToStatePage = page => {
-    window.scrollTo(0, 0);
-    history.push(page);
-  };
+  const goToStatePage = React.useCallback(
+    page => {
+      window.scrollTo(0, 0);
+      history.push(page);
+    },
+    [history],
```

Comment:
This might not work as intended, because the [history object](https://reactrouter.com/web/api/history) is a mutable object which makes detecting changes to it a bit unpredictable. We might not need this function anyways, so that's probably okay (we do listen to `history.location` in other places)

Reply:
Thanks for the flag.  I think it's okay if it mutates... it just means we'll re-generate this callback, which isn't harmful (except it causes the map to rerender unnecessarily, but that's okay as long as it's not frequent).